index.js

```diff
@@ -19,25 +19,15 @@ TranspilerFilter.prototype.extensions = ['js'];
 TranspilerFilter.prototype.targetExtension = 'js';
 
 TranspilerFilter.prototype.processString = function (string, relativePath) {
-  var result;
-  var opts = this.options;
-  opts.filename = opts.sourceMapName = opts.sourceFileName = relativePath;
+  var opts = this.copyOptions();
 
-  try {
-    result = transpiler.transform(string, opts);
+  opts.filename = opts.sourceMapName = opts.sourceFileName = relativePath;
 
-    /*
-     * At this time, an array of source maps can be accessed by using
-     * `result.map`, but I don't know how to write them out as files.
-     *
-     * Thus `sourceMap: true` has no effects yet, if you know how to solve this,
-     * send a pull request in advance.
-     */
+  return transpiler.transform(string, opts).code;
+};
 
-    return result.code;
-  } catch (err) {
-    throw err;
-  }
+TranspilerFilter.prototype.copyOptions = function() {
+  return JSON.parse(JSON.stringify(this.options));
```

Comment:
I think this is fine for now, once the future lands we can do that.

If you are extremely set on pulling in clondeep for this, I will doit.